config/templates/deployment_patch.yaml.in

```diff
@@ -24,12 +24,12 @@ spec:
       containers:
         - name: cockroach-operator
           args:
-            - feature-gates
-            - Upgrade=false,ResizePVC=true
+            - -skip-webhook-config
```

Comment:
Should we have this skipped by default?

Reply:
I don't think so. We only want to skip this in OpenShift